java/java.editor/src/org/netbeans/modules/java/editor/codegen/LoggerGenerator.java

```diff
@@ -152,15 +164,66 @@ public void run(WorkingCopy copy) throws IOException {
         }
     }
 
+    public static String getBaseLoggerName() {
+        // // Undocumented feature/property subject to change/removal. Need a UI.
+        // String name = System.getProperty("LOGGER_GENERATOR_BASE_LOGGER_NAME");
+        // return name == null ? "LOG" : name;
+        return "LOG";
+    }
```

Comment:
I can accept, that this akin to the tab-vs-space or "the right IDE" wars, but if we want to parametrize this, should this be project level configurable?

And for the wording I would suggest:

```suggestion
    public static String getBaseLoggerName() {
        // Preparation for a potential future configurability of the loggers field name
        return "LOG";
    }
```

Reply:
> if we want to parametrize this, should this be project level configurable?

If there's follow up work, I'd be inclined to add it to the rest of logging configuration
```
...java.hints.errors.ErrorFixesFakeHintSURROUND_WITH_TRY_CATCH
```
which makes it project configurable. Probably rename it to indicate it's general logging stuff, and include additional stuff as needed.

Fixed comment.